ts/components/wallet/WalletLayout.tsx

```diff
@@ -0,0 +1,150 @@
+import { Body, Container, Content, Text, View } from "native-base";
+import { Button } from "native-base";
+import { Left } from "native-base";
+import * as React from "react";
+import { ScrollView } from "react-native";
+import { Image, StyleSheet, TouchableHighlight } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import Icon from "../../theme/font-icons/io-icon-font";
+import variables from "../../theme/variables";
+import { WalletStyles } from "../styles/wallet";
+import AppHeader from "../ui/AppHeader";
+
+const styles = StyleSheet.create({
+  darkGrayBg: {
+    backgroundColor: variables.brandDarkGray
+  },
+  whiteBg: {
+    backgroundColor: variables.colorWhite
+  },
+  noBottomPadding: {
+    paddingBottom: 0
+  }
+});
+
+export enum CardType {
```

Comment:
add comment to this enum, what does it describe? also describe each value - what is a `FAN` `CardType`?